roles/common/tasks/rbac_setup.yml

```diff
@@ -11,6 +11,9 @@
     url_password: "{{mds_super_user_password}}"
     force_basic_auth: true
   register: cluster_id_query
+  until: cluster_id_query.status == 200
+  retries: 5
+  delay: 10
```

Comment:
Have we validated this is the correct amount of time to wait?  I only ask as this could sit here for 50 seconds, which seems quite long to me.

Reply:
50 seconds is the timeout. If the service is up and running, it could be as low as 5 seconds. 
We have tried the tests `rbac-plain-provided-debian` which was failing for different components intermittently. The is same is successful in [this run](https://jenkins.confluent.io/job/cp-ansible-on-demand/131/). If you see the logs, the retry did play a role and the tasks ended up after 2-3 retries

Reply:
Sounds good.